contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -13,50 +13,42 @@ import "../libraries/CommonLibrary.sol";
 import "../libraries/external/FullMath.sol";
 import "../utils/DefaultAccessControl.sol";
 import "@prb/math/contracts/PRBMathSD59x18.sol";
-import "hardhat/console.sol";
+import "../interfaces/utils/ILpCallback.sol";
 
-contract LPOptimiserStrategy is DefaultAccessControl {
+contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
     using SafeERC20 for IERC20;
 
     // IMMUTABLES
     address[] public _tokens;
     IERC20Vault public immutable _erc20Vault;
 
     // INTERNAL STATE
-
     IVoltzVault internal _vault;
     uint256[] internal _pullExistentials;
     IMarginEngine internal _marginEngine;
     IPeriphery internal _periphery;
     IVAMM internal _vamm;
-    IVoltzVault.TickRange internal _currentPosition;
 
     // MUTABLE PARAMS
-
     uint256 internal _sigmaWad; // y (standard deviation parameter in wad 10^18)
-    int256 internal _max_possible_lower_bound_wad; // should be in fixed rate
-
+    int256 internal _maxPossibleLowerBoundWad; // should be in fixed rate
```

Comment:
can change the comment on the right to smth more descriptive, e.g. "Maximum Possible Fixed Rate Lower bounds when initiating a rebalance", will make auditing the code more straightforward